### PR TITLE
Remove asap ready-by option

### PR DESCRIPTION
**Pull request by jknapp25**
For test details [see here](https://www.notion.so/photons/Remove-ASAP-option-67a0fd490acd43eb8a8f224563709995)

image

### CODE REVIEW

File: apps/patient/src/views/ReadyBy.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const ReadyBy = () => {
 
           <VStack spacing={3} w="full">
             {t.readyBy.options.map((option, i) => {
-              const isDisabled = i !== (0 | 6) && currentTimeIsAfterOption(option);
+              const isDisabled = i !== 6 && currentTimeIsAfterOption(option);
```
Comment (captDaylight) [inline apps/patient/src/views/ReadyBy.tsx:59]: better way to do this than using a hard coded index?

Comment (jknapp25) [reply apps/patient/src/views/ReadyBy.tsx]: fixed

File: apps/patient/src/views/ReadyBy.tsx [hunk 2]
```diff
@@ -11,7 +11,8 @@ import { PoweredBy } from '../components/PoweredBy';
 import t from '../utils/text.json';
 import { OrderContext } from './Main';
 
-const currentTimeIsAfterOption = (option: string): boolean => {
+const checkDisabled = (option: string): boolean => {
+  if (option === 'tomorrow') return false;
```
Comment (captDaylight) [inline apps/patient/src/views/ReadyBy.tsx:15]: will this catch if the text is "Tomorrow"? is it lowercased somewhere?

Comment (jknapp25) [reply apps/patient/src/views/ReadyBy.tsx]: ah interesting, works without upper case, removing

**Pull request merged**